.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
       - uses: actions/setup-node@v1
         with:
-          node-version: "12"
+          node-version: "14"
```

Comment:
May as well bump this to 16? Since that's the current LTS

Reply:
```suggestion
          node-version: "16"
```